roles/ruby_s/tasks/main.yml

```diff
@@ -0,0 +1,65 @@
+---
+- name: ruby | check version
+  ansible.builtin.shell: ruby -v | awk '{print $2}'
+  register: installed_ruby
+  ignore_errors: true
```

Comment:
Ignoring errors here seems risky. What kind of errors cropped up when you ran this?